assets/src/edit-story/components/panels/design/pageAttachment/pageAttachment.js

```diff
@@ -108,16 +107,39 @@ function PageAttachmentPanel() {
   );
 
   const isDefault = _ctaText === defaultCTA;
+  const showTextInput = Boolean(url) && !isInvalidUrl;
+
+  const handleChange = useCallback(
+    ({ target }) => {
+      const { value } = target;
+      // This allows smooth input value change without any lag.
+      _setCtaText(value);
+      debouncedCTAUpdate(value);
+    },
+    [debouncedCTAUpdate]
+  );
+
+  const handleBlur = useCallback(
+    ({ target }) => {
+      if (!target.value) {
+        updatePageAttachment({ ctaText: defaultCTA });
+        _setCtaText(defaultCTA);
+      } else {
+        cancelCTAUpdate();
+        updatePageAttachment({
+          ctaText: _ctaText ? _ctaText : defaultCTA,
+        });
+      }
+    },
+    [_ctaText, cancelCTAUpdate, defaultCTA, updatePageAttachment]
+  );
+
   return (
     <SimplePanel
       name="pageAttachment"
       title={__('Page Attachment', 'web-stories')}
     >
       <LinkInput
-        description={__(
-          'Type an address to add a page attachment',
-          'web-stories'
-        )}
         onChange={(value) => updatePageAttachment({ url: value })}
         onFocus={onFocus}
         value={url || ''}
```

Comment:
There's something funky going on around here when multiple elements are selected with different link targets. When you select the elements, the input correctly says "Mixed", but if you focus and blur the input without entering anything, the input switches to `"http://((MULTIPLE))"` which I'm pretty sure is wrong.

![Screen Shot 2021-03-04 at 23 33 12](https://user-images.githubusercontent.com/637548/110067846-83f33600-7d42-11eb-9782-b36bb2166007.png)

However, I'm inclined to say that we just merge this as-is and we fix all the Mixed stuff in a single follow-up PR "later" (still in this release), as we're doing it in many different ways in different PR's.

Reply:
The multi-value handling was not implemented to the link input indeed... For the Page Attachment panel it actually doesn't matter since only one Page can be selected at a time.